--- Mowitz/Makefile.am.link	2011-01-03 07:34:28.000000000 +0000
+++ Mowitz/Makefile.am	2011-01-03 07:35:03.000000000 +0000
@@ -65,6 +65,7 @@
 	$(srcdir)/mkt1cfg $(pkgdatadir) > t1lib.config
 
 libMowitz_la_LDFLAGS = -version-info 3:0:3
+libMowitz_la_LIBADD = $(X_LIBS)
 
 INCLUDES = $(DEBUG) @X_CFLAGS@ \
 	-DNeedFunctionPrototypes -I.. \
--- Mowitz/Makefile.in.link	2011-01-03 07:34:38.000000000 +0000
+++ Mowitz/Makefile.in	2011-01-03 07:35:17.000000000 +0000
@@ -162,7 +162,7 @@
 CONFIG_CLEAN_FILES =
 LTLIBRARIES = $(lib_LTLIBRARIES)
 
-libMowitz_la_LIBADD =
+libMowitz_la_LIBADD = $(X_LIBS)
 am_libMowitz_la_OBJECTS = MwApplicationShell.lo MwCanvas.lo MwCheck.lo \
 	MwCheckME.lo MwCombo.lo MwDialog.lo MwFilesel.lo MwFrame.lo \
 	MwGcs.lo MwHandle.lo MwImage.lo MwListTree.lo MwNotebook.lo \
